source/quickstart.txt

```diff
@@ -247,6 +247,15 @@ documentation provides details on using the following endpoints:
    * - :ref:`c2c-api-reverse`
      - Reverses the direction of a committed sync operation.
 
+Finalize Cutover Process
+------------------------
```

Comment:
Now that we've fixed this, could you replace the build log link in the PR description? Thanks!

Reply:
Sorry about that, just fixed!